helpers.go

```diff
@@ -316,28 +319,53 @@ func (s *Spec) GetTestDeps(targetKey string) []string {
 	return out
 }
 
-func (s *Spec) GetSymlinks(target string) map[string]SymlinkTarget {
-	lm := make(map[string]SymlinkTarget)
-
-	if s.Image != nil && s.Image.Post != nil && s.Image.Post.Symlinks != nil {
-		for k, v := range s.Image.Post.Symlinks {
-			lm[k] = v
+func (s *Spec) GetImagePost(target string) *PostInstall {
+	img := s.Targets[target].Image
```

Comment:
is there any change that the arg `target` is not existing withing `s.Targets`? If so maybe we should check before using `.Image`?

Reply:
`s.Targets[target]` is never nil here.